subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -370,490 +370,64 @@ Once a configuration is resolved it is immutable. Changing its state or the stat
 
 To learn more about the API of the configuration class see the API documentation: api:org.gradle.api.artifacts.Configuration[].
 
-[[sec:repositories]]
-=== Repositories
+[[sec:declaring_repositories]]
+=== Declaring repositories
```

Comment:
Should we consider "Declaring Repositories" as a separate chapter? I think it would be _much_ more discoverable than buried in the Advanced Dependency Management chapter.

Reply:
In the long run we'll definitely want to break up the whole dependency management chapter. I am just wondering when it is a good time to do so. Optimally that would be the case if we have a consistent outline/structure in the current chapter. Imagine we'd just break out "declaring dependencies" and "declaring repositories" right now then I'd think that the rest would get rather inconsistent. Maybe we can make the overall flow better and then break it up. Would you agree?

Reply:
I would recommend we prioritize discoverability over "flow" because users do not read chapters and sections in order... but you're in charge of this chapter so I'll let you make the call here, as long as we get this settled by Gradle 4.6 I think.